packages/api/endpoints/elasticsearch.js

```diff
@@ -1,233 +0,0 @@
-    .then((response) => response.body);
-
-  if (!destExists) {
-    try {
-      await createIndex(esClient, newIndex);
-      log.info(`Created destination index ${newIndex}.`);
-    } catch (error) {
-      return res.boom.badRequest(`Error creating index ${newIndex}: ${error.message}`);
-    }
-  }
-
-  try {
-    await esClient.client.indices.updateAliases({
-      body: {
-        actions: [
-          { remove: { index: currentIndex, alias: aliasName } },
-          { add: { index: newIndex, alias: aliasName } },
-        ],
-      },
-    });
-
-    log.info(`Removed alias ${aliasName} from index ${currentIndex} and added alias to ${newIndex}`);
-  } catch (error) {
-    return res.boom.badRequest(
-      `Error removing alias ${aliasName} from index ${currentIndex} and adding alias to ${newIndex}: ${error}`
-    );
-  }
-
-  let message = `Change index success - alias ${aliasName} now pointing to ${newIndex}`;
-
-  if (deleteSource) {
-    await esClient.client.indices.delete({ index: currentIndex });
-    log.info(`Deleted index ${currentIndex}`);
-    message = `${message} and index ${currentIndex} deleted`;
-  }
-
-  return res.send({ message });
-}
-
-async function indicesStatus(req, res) {
-  const esClient = await getEsClient();
-
-  return res.send(await esClient.client.cat.indices({}));
-}
-
-async function indexFromDatabase(req, res) {
-  const esClient = await getEsClient();
-  const indexName = req.body.indexName || timestampedIndexName();
-  const { postgresResultPageSize, postgresConnectionPoolSize, esRequestConcurrency } = req.body;
-
-  await createIndex(esClient, indexName)
-    .catch((error) => {
-      if (!(error instanceof IndexExistsError)) throw error;
-    });
-
-  const asyncOperationId = uuidv4();
-  const asyncOperationEvent = {
-    asyncOperationId,
-    callerLambdaName: getFunctionNameFromRequestContext(req),
-    lambdaName: process.env.IndexFromDatabaseLambda,
-    description: 'Elasticsearch index from database',
-    operationType: 'ES Index',
-    payload: {
-      indexName,
-      reconciliationReportsTable: process.env.ReconciliationReportsTable,
-      esHost: process.env.ES_HOST,
-      esRequestConcurrency: esRequestConcurrency || process.env.ES_CONCURRENCY,
-      postgresResultPageSize,
-      postgresConnectionPoolSize,
-    },
-  };
-
-  log.debug(`About to invoke lambda to start async operation ${asyncOperationId}`);
-  await startAsyncOperation.invokeStartAsyncOperationLambda(asyncOperationEvent);
-  return res.send({ message: `Indexing database to ${indexName}. Operation id: ${asyncOperationId}` });
-}
-
-async function getCurrentIndex(req, res) {
-  const esClient = await getEsClient();
-  const alias = req.params.alias || defaultIndexAlias;
-
-  const aliasIndices = await esClient.client.indices.getAlias({ name: alias })
-    .then((response) => response.body);
-
-  return res.send(Object.keys(aliasIndices));
-}
-
-// express routes
-router.put('/create-snapshot', createEsSnapshot);
```

Comment:
Is there an argument for keeping any of this? Specifically the snapshot endpoint? Would this be useful for DAACs that still have an orphan ES instance?

Reply:
I don't know. If anything should be kept and we have a reason, that seems fine. But I don't know if any should be and I was just going with "remove all the ES things" except for what is needed for CloudMetrics and the other tickets I wrote. Please let me know if there's anything that should be put back.

Reply:
the functionality for the endpoint was never actually implemented, so keeping this would not do anything.